scep-ops.cmd

```diff
@@ -0,0 +1,34 @@
+@if not defined _echo @echo off
+
+set NO_DASHES_ARG=%1
+if not defined NO_DASHES_ARG goto no_args
+if /I [%NO_DASHES_ARG:-=%] == [?] goto Usage
+if /I [%NO_DASHES_ARG:-=%] == [h] goto Usage
+
+if not defined NO_DASHES_ARG goto Usage
+if /I [%NO_DASHES_ARG:-=%] == [i] (
+  echo Instaling SCEP ...
+  call \\ddsccmps\Client\scepinstall.exe /q /s /policy \\ddsccmps\Client\ep_defaultpolicy.xml
```

Comment:
I don't like this dependency on a random file share.

Reply:
Maybe put it in a package or something like we handle other dependencies / utilities.  This still seems too fragile and like a major point of failure.  Packages we've invested in and do things like retry.

Reply:
Also, getting the latest changes isn't always desirable.  We've been bitten many times in the past by not having control over dependencies when an external team decides to make a change.  You might ask if they already provide a nupkg.  I know DDIT has pointed me at tools in the past and then after some investigation I've found that they actually have a nupkg for the tool on an internal feed.